tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/openwire/interop/GeneralInteropTest.java

```diff
@@ -243,17 +243,60 @@ public void testFailoverReceivingFromCore() throws Exception {
          assertEquals(text + 1, ((TextMessage)message).getText());
          message.acknowledge();
 
+         System.err.println("id: " + message.getJMSMessageID());
```

Comment:
logger.debug

is this by accident?

I had recently (an year or so ago) removed all System.outs and System.errs from the test suite.

Reply:
good catch. removed that. thanks!